getting-started/overview/system-requirements.rst

```diff
@@ -7,8 +7,8 @@ Browser compatibility
 ---------------------
 
 Archivematica is tested most extensively in Firefox and Chrome. There have been
-known issues in testing with Internet Explorer (see Bug
-`#7246 <https://projects.artefactual.com/issues/7246>`_). Minimal, but successful,
+known issues in testing with Internet Explorer (see Bug `#7246
+<https://projects.artefactual.com/issues/7246>`_). Minimal, but successful,
```

Comment:
Can you change this to use a hyperlink target, rather than an inline link? See the [style guide](https://github.com/artefactual/archivematica-docs/blob/12da09e3664025169450ac8b913639ff85ebba84/style_guide.md#external-links) for more info!